src/consola.js

```diff
@@ -8,7 +9,7 @@ class Consola {
   constructor (options = {}) {
     this._reporters = options.reporters || []
     this._types = options.types || Types
-    this.level = options.level !== undefined ? options.level : 3
+    this.level = normalizeLogLevel(options.level, this._types)
```

Comment:
We would convert `level` to a getter/setter to also normalize when assigning literal to `consola.level` dynamically (eg: https://github.com/nuxt/framework/blob/3d258d304e2e6e5e75e3a239796321e2823fb8ef/packages/test-utils/src/nuxt.ts#L57)